servermaster/resource/capacity_impl.go

```diff
@@ -87,7 +89,8 @@ func (m *CapRescMgr) allocateTasksWithNaiveStrategy(
 		// No resources in this cluster
 		return false, nil
 	}
-	var idx int = 0
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+	idx := r.Intn(len(resources))
```

Comment:
What about storing a `*rand.Rand` field in `CapRescMgr`, then we don't need to create a new seed every time.